### ISSUE TITLE
Create new Utterance for "activation" alerts

### ISSUE THREAD

**Issue by jessegreenberg** [phetsims/scenery-phet#492]
The changes in https://github.com/phetsims/scenery-phet/issues/491 are generally good for most utterances, but there are a category of Utterances that will need a longer `alertStableDelay` so that alerts generally don't get repeated when you press and hold down buttons. We are seeing now that if you press and hold the ResetAllButton, you will get two "reset" alerts because the `alertStableDelay` is longer than the default press and hold delay for activation with the "enter" key.

Potential names:
- ActivationUtterance
- ActivateUtterance
- ButtonUtterance
- ClickUtterance
- InteractionUtterance
- ActionUtterance
- SlowUtterance
- LongDelayUtterance
- StableUtterance
- LongStableUtterance
- PressUtterance

I like ActivationUtterance the best because it aligns with the W3C term for pressing things. I also like ButtonUtterance and ClickUtterance. But maybe we will want this for more than buttons and clicks.

**Comment by jessegreenberg** [comment]
Created and used in ResetAllButton in the above commit.

**Comment by jessegreenberg** [comment]
Also used in NavigationBarSoundToggleButton.

I inspected instances of `new Utterance` and didn't see any other cases right now that would benefit from `ActivationUtterance`.

@zepumph could you please review? Mostly, are you OK with the name ActivationUtterance?

**Comment by jessegreenberg** [comment]
Following a similar pattern, we are going to create a special type of Utterance that will assist with alerts that happen after a value changes called ValueChangeUtterance.

**Comment by jessegreenberg** [comment]
Done, and ready for review again.

**Comment by zepumph** [comment]
@jessegreenberg, above I updated ISLC suite sims to use `ActivationUtterance` and `ValueChangeUtterance`. This is looking pretty nice. I really like that we have factored out control of timing to a single place based on use. I'm not totally convinced that 1000ms is the right time for the value change utterance. To me it is a bit long, but I defer to what the designed decision is. 

@terracoda could you please review the timing of alerts in GFLB. They just changed from the above commits, and I think your thoughts on them could inform the final outcome of this issue. Basically checkbox alerts will be a bit longer (~half a second as an "Activation Utterance"), and position/mass changes will be a fair bit longer (~one second as a "Value Change Utterance"). Does this work for GFLB? Do you think that these would work in general for most sims?

**Comment by jessegreenberg** [comment]
I agree, it is a bit longer than ideal. We landed on 1000 ms in https://github.com/phetsims/john-travoltage/issues/315 because we discovered that JAWS has a hard time reading alerts if they happen too quickly after reading aria-valuetext. I added a missing reference link in the doc. If we want to reduce the time, we will want to test changes against JAWS and VO.

**Comment by terracoda** [comment]
@jessegreenberg when I worked with sim on Friday, and the alerts seemed slow. I will review again, and post here on Monday.

**Comment by terracoda** [comment]
@jessegreenberg, the alerts feel like they appear in the orange box a little slowly when using the A11y View, but I think they sound fine when I use the sim with VoiceOver.

**Comment by terracoda** [comment]
@jessegreenberg, I'm going to close this issue. Pleas ere-open if I closed prematurely.
